### ISSUE TITLE
ETS2 Local Radio Didn't detect ETS2

### ISSUE THREAD

**Issue by VoXaN24** [Koenvh1/ets2-local-radio#157]
I have the error "0000007E" With Visual C++ Redistributable for Visual Studio 2015 installed,
I have the last non-beta version
I have joined the Log of ETS2
[game.log.txt](https://github.com/Koenvh1/ets2-local-radio/files/7070398/game.log.txt)
![image](https://user-images.githubusercontent.com/30030208/131211913-d4167ed5-6131-4ca7-8e7d-10993e8cd7a8.png)

**Comment by BendoH69** [comment]
Yep, having the exact same issue with no solution, but 1 year later than you. :P
